make test fails

Despite us not having any tests yet, make test / go fmt fails:

```
▶ make test
go get github.com/mattn/goveralls
go get github.com/modocache/gover
go get github.com/Masterminds/glide
go get github.com/sgotti/glide-vc
go get github.com/golang/lint/golint
./scripts/check-vendor.sh
Checking for nested vendor dirs
OK
Checking if vendor was cleaned using glide-vc.
OK
./scripts/check-gofmt.sh
gofmt ERROR - These files are not formated by gofmt:
./cmd/deploy.go
./cmd/generate.go
./pkg/encoding/fix.go
./pkg/transform/kubernetes/kubernetes.go
Makefile:44: recipe for target 'gofmt' failed
make: *** [gofmt] Error 1
```

--- a/cmd/deploy.go
+++ b/cmd/deploy.go
@@ -20,8 +20,8 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/spf13/cobra"
 	pkgcmd "github.com/kedgeproject/kedge/pkg/cmd"
+	"github.com/spf13/cobra"
 )
 
 // Variables

--- a/cmd/generate.go
+++ b/cmd/generate.go
@@ -20,8 +20,8 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/spf13/cobra"
 	pkgcmd "github.com/kedgeproject/kedge/pkg/cmd"
+	"github.com/spf13/cobra"
 )
 
 var (

--- a/pkg/encoding/fix.go
+++ b/pkg/encoding/fix.go
@@ -20,8 +20,8 @@ import (
 	"fmt"
 	"strconv"
 
-	"github.com/pkg/errors"
 	"github.com/kedgeproject/kedge/pkg/spec"
+	"github.com/pkg/errors"
 )
 
 func fixApp(app *spec.App) error {

--- a/pkg/transform/kubernetes/kubernetes.go
+++ b/pkg/transform/kubernetes/kubernetes.go
@@ -23,8 +23,8 @@ import (
 	"strings"
 
 	"github.com/davecgh/go-spew/spew"
-	"github.com/pkg/errors"
 	"github.com/kedgeproject/kedge/pkg/spec"
+	"github.com/pkg/errors"
 	"k8s.io/client-go/pkg/api"
 	"k8s.io/client-go/pkg/api/resource"
 	"k8s.io/client-go/pkg/runtime"

